meta/src/grammar.pest

```diff
@@ -7,11 +7,12 @@
 // option. All files in the project carrying such notice may not be copied,
 // modified, or distributed except according to those terms.
 
-grammar_rules = _{ SOI ~ grammar_rule+ ~ EOI }
+grammar_rules = _{ SOI ~ grammar_doc* ~ (grammar_rule)+ ~ EOI }
 
 grammar_rule = {
```

Comment:
maybe it'll be good to "dogfood" this change in the meta grammar and use it to add doc comments, but that could be done in a follow up

Reply:
I could possibly do that in a follow up PR